src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Card from './components/card/card.component'
+
+import './App.css';
+
+class App extends Component {
+  state = {
+    
+  }
+
+  render() {
+    
+    return (
+      <div>
+        <nav className='nav'>
+          <h2 className='wrapper'>InnRoad</h2>
+        </nav>
+        <div className='wrapper'>
+          <Card />
+        </div>
+      </div>
+    );
+  }
+
+}
```

Comment:
Convert this to functional component instead since you are not using any state here.